src/pages/view-test-result/cat-a-mod2/components/debrief-card/__tests__/debrief-card.spec.ts

```diff
@@ -0,0 +1,265 @@
+import { ComponentFixture, async, TestBed } from '@angular/core/testing';
+import { IonicModule, Config } from 'ionic-angular';
+import { DebriefCardComponent } from '../debrief-card';
+import { ConfigMock } from 'ionic-mocks';
+import { MockComponent } from 'ng-mocks';
+import { TickIndicatorComponent } from '../../../../../../components/common/tick-indicator/tick-indicator';
+import {
+  DangerousFaultBadgeComponent,
+} from '../../../../../../components/common/dangerous-fault-badge/dangerous-fault-badge';
+import {
+  SeriousFaultBadgeComponent,
+} from '../../../../../../components/common/serious-fault-badge/serious-fault-badge';
+import {
+  DrivingFaultsBadgeComponent,
+} from '../../../../../../components/common/driving-faults-badge/driving-faults-badge';
+import { DataRowComponent } from '../../../../../../components/common/data-row/data-row';
+import { DataRowCustomComponent } from '../../../../../../components/common/data-row-custom/data-row-custom';
+import { DataRowWithListComponent } from '../../../../components/data-row-with-list/data-list-with-row';
+import { FaultsDataRowComponent } from '../../../../components/faults-data-row/faults-data-row';
+import { VehicleChecksDataRowComponent } from '../../../../components/vehicle-checks-data-row/vehicle-checks-data-row';
+import { FaultSummaryProvider } from '../../../../../../providers/fault-summary/fault-summary';
+import { FaultCountProvider } from '../../../../../../providers/fault-count/fault-count';
+// todo: PREP-AMOD2 change to CatAmod2UniqueTypes when schema changes are ready
+import { CatBEUniqueTypes } from '@dvsa/mes-test-schema/categories/BE';
+import {
+  DataRowListItem,
+  TestRequirementsLabels,
+  ViewTestResultLabels,
+} from '../../../../components/data-row-with-list/data-list-with-row.model';
+import { manoeuvreTypeLabels } from '../../../../../../shared/constants/competencies/catbe-manoeuvres';
```

Comment:
A todo is missing for here

Reply:
Thanks Szabi, this also helped highlight there were 2 missing `__tests__` folders I'd neglected.